src/vscode-bundle.js

```diff
@@ -18,44 +18,15 @@
 const rimraf = require('../vscode/node_modules/rimraf');
 const vfs = require('../vscode/node_modules/vinyl-fs');
 const ext = require('../vscode/build/lib/extensions');
-const { theiaExtension, extensions, run, vscodeExtensions } = require('./paths.js');
-
-const fs = require('fs');
-const path = require('path');
+const { theiaExtension, extensions, run } = require('./paths.js');
 
 rimraf.sync(extensions());
 (async () => {
     await new Promise((resolve, reject) => {
-        ext.packageLocalExtensionsStream(false)
+        ext.packageLocalExtensionsStream(false, false)
             .pipe(vfs.dest(theiaExtension()))
             .on('error', reject)
             .on('end', resolve);
     });
-    copyYarnLock(vscodeExtensions(), extensions())
     await run('yarn', ['install', '--production'], extensions('emmet'));
 })();
-
-/**
- * The 'yarn.lock' file is now required by the 'vsce' cli.
- * This method copies it from the given source folder to a corresponding
- * folder in the target.
- * 
- * The method assumes that for each subfolder in the target a corresponding
- * subfolder with the same name exists in the source folder.
- * 
- * @param {string} sourceDir - The path of the source folder.
- * @param {string} targetDir - The path of the destination folder.
- */
-function copyYarnLock(sourceDir, targetDir) {
```

Comment:
Do we not need a package-lock.json instead?

Reply:
As far as I could see, vsce tool does not need any package-lock.json. It was used as far as I can see to detect the kind of tool used, but it is ignored in the produced vsix. 
I did a side-by-side compare of the 2 generated vsix extensions between 1.88.1 and 1.94.2. I do not see any differences between the 2 with the current configuration, especially comparing typescript-language-feature and emmet, that have some specific handling during build and packaging process.